src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
```

Comment:
Get all properties right away
```suggestion
export const Person = ({ person: { name, ... } }) => {
```